packages/slice-machine/package.json

```
@@ -49,7 +49,7 @@
     "@prismicio/mock": "0.3.3",
     "@prismicio/mocks": "2.4.0",
     "@prismicio/simulator": "0.1.4",
-    "@prismicio/types-internal": "2.8.0",
+    "@prismicio/types-internal": "2.9.0-alpha.0",
```

To be applied when package is released

```suggestion
    "@prismicio/types-internal": "2.9.0",
```